guides/installation/setups/docker.md

```diff
@@ -8,116 +8,263 @@ nav:
+```bash
+docker compose ps
+```
+“Healthy” means the service passed its internal health check and is ready to use.
+</details>
+
+
+Once the containers are running, you can install Shopware in one of two ways:
+- **Browser installer**: open <http://localhost:8000> to walk through the installation wizard.
+- **CLI**: run the following command to perform a quick, non-interactive setup:
 
 ```bash
 make setup
 ```
 
-This will install Shopware itself, create an admin user with username `admin` and password `shopware`.
+Both methods install Shopware and prepare your environment. The CLI setup automatically creates the database and an admin user with username `admin`, password `shopware`.
 
 :::info
-For the Database Configuration step, you should put the host as `database` instead of `localhost`, which is the Docker container name.
+If you are installing inside the Docker containers (the default when using `make up` and `make setup`), set the database host to `database`. This is the internal service name of the MariaDB container on the Docker network. Inside the containers, `localhost` would refer only to the container itself, not to the database.
+
+If you connect to the database from your host machine (for example, via Adminer or a local MySQL client), use 127.0.0.1 or `localhost` and the exposed port shown in `docker compose ps`.
 :::
 
-If you want to stop the setup, you can run `make stop` and to start it again, you can run `make up` again. If you want to remove the containers, you can run `make down`. This will remove all containers and **keep the data**. If you want to remove all containers and the data, you can run `docker compose down -v`
+<details>
+<summary>🔑 Access key explained (click to expand)</summary>
+
+During setup, you’ll see an output similar to this:
+
+```
+Access tokens:
++------------+----------------------------+
+| Key | Value |
++------------+----------------------------+
+| Access key | `string of capital letters` |
+```
+
+This **access key** is automatically generated for your default **Sales Channel** (usually *Storefront*). It's used for authenticating requests to the [Store API](../../resources/references/store-api-reference.md)—for example, when fetching product or category data from an external app, headless storefront, or API client.
+
+Example usage:
+
+```bash
+curl -H "sw-access-key: YOUR_ACCESS_KEY" \
+     http://localhost:8000/store-api/product
+```
+
+You can view or regenerate this key later in the Admin under Sales Channels → [Your Channel] → API Access.
+
+💡 **Tip:** The access key is not for logging in to the Admin. It’s for programmatic access to your storefront’s data via the Store API.
+</details>
+
+If you want to stop the setup, run `make stop`. 
+
+To start it again, use `make up`. 
+
+If you want to remove the containers, run `make down`. This removes all containers but **keeps your data**. 
+
+To remove both containers *and* data, run `docker compose down -v`. You can then do a full reset.
```

Comment:
`make down` is executing a docker compose down internally, so you can execute `make down` in this case as well 

and exactly how to do a full reset is missing, we should either give that information or skip that part